Source/dict.h

```diff
@@ -32,20 +32,23 @@
 #ifndef DICT_LIST_H
 #define DICT_LIST_H
 
-typedef void *DictKey;
+#include "../Include/tesselator.h"
+#include "mesh.h"
+#include "sweep.h"
```

Comment:
sweep.h includes dict.h
Let's forward declare the types instead, they are used as pointers here.

Reply:
Yeah, I think I messed up something with the includes here in any case.